jujugui/templates/index.html.mako

```diff
@@ -299,15 +299,15 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
     <script src="${convoy_url}?app/assets/javascripts/version.js"></script>
     <!-- data-manual tells the Prism syntax highlighting lib to not auto-highlight -->
     <!-- The below init js is for the transition from app.js -->
-    <!-- <script src="${convoy_url}?app/init-pkg.js"></script> -->
+    <script src="${convoy_url}?app/init-pkg.js"></script>
```

Comment:
These changes will also need to be made to the index.go once the bugs are ironed out and unnecessary dependencies removed.